readthedocs/settings/dev.py

```diff
@@ -31,7 +31,7 @@ def DATABASES(self):  # noqa
     SLUMBER_API_HOST = 'http://localhost:8000'
     PUBLIC_API_URL = 'http://localhost:8000'
 
-    BROKER_URL = 'redis://localhost:6379/0'
+    CELERY_BROKER_URL = 'redis://localhost:6379/0'
```

Comment:
So I think this will fail the tests!
Can you remove this and keep it like previous?